src/components/LoansByCategory/QuickFilters/CheckboxList.vue

```diff
@@ -75,13 +75,16 @@ export default {
 				const index = this.selected.indexOf(item.value);
 				const exists = index !== -1;
 				if (isAll) {
-					if (exists) this.selected.splice(index, 1);
-				} else if (!exists) this.selected.push(item.value);
+					if (exists) {
+						this.updateSelected(this.selected, item.value);
```

Comment:
It looks like the component is emitting an event for each item that was selected when select all is toggled. There should only be one event emitted for a user action.

Reply:
Yeah, this is b/c the updater on parent was planned for one event. Selecting all is the less probable action here, main idea is to select a country you are interested.
We need to launch this asap, but if you believe a refactor is needed for this experiment I can fix it.

Reply:
Gotcha. If it works alright in functional testing, please add a `TODO` here to refactor into one emitted event. In general you really only want one distinct event per user action, unless there's a chain of actions. Here we have one user interaction with one result (reset filter).

Reply:
Good! Agree, I added a new ticket for this https://kiva.atlassian.net/browse/CORE-820

Reply:
Thanks!